superset-frontend/src/components/Chart/DrillBy/DrillByMenuItems.tsx

```diff
@@ -170,7 +172,9 @@ export const DrillByMenuItems = ({
                 <Menu.Item
                   key={`drill-by-item-${column.column_name}`}
                   {...rest}
-                  onClick={openModal}
+                  onClick={openModal.bind(
+                    column.verbose_name || column.column_name,
```

Comment:
Can we pass the whole `column` object to the modal? We need both `verbose_name` (to display in breadcrumbs) and `column_name` (to apply as `groupby`)